aten/src/ATen/native/native_functions.yaml

```diff
@@ -849,10 +860,12 @@
   matches_jit_signature: True
   device_guard: False
 
-- func: empty_like(Tensor self, *, TensorOptions options) -> Tensor
+- func: empty_like(Tensor self, *, ScalarType dtype, Layout layout, Device device) -> Tensor
```

Comment:
this look wrong -- doesn't this mean you have to provide values for dtype, layout, device?

Reply:
Even more so, how can a keyword argument "options" not require a default value?

Reply:
If you do provide this a default value, it would, logically, make sense to provide one for the generated C++ code, however, you'll then get conflicts with empty_like(Tensor self) and empty_like(Tensor self, TensorOptions options={}), because it's ambiguous which symbol to use.

Reply:
ah, ok, the issue is this isn't combined with the above definition.  Do you know why that is?